program/tests/helpers/utils.rs

```diff
@@ -0,0 +1,76 @@
+use {
+    mollusk_svm::Mollusk,
+    solana_account::{Account, AccountSharedData, WritableAccount},
+    solana_instruction::Instruction,
+    solana_pubkey::Pubkey,
+    solana_rent::Rent,
+    solana_stake_interface::{stake_history::StakeHistory, state::StakeStateV2},
+    solana_sysvar_id::SysvarId,
+    solana_vote_interface::state::{VoteStateV4, VoteStateVersions},
+    std::collections::HashMap,
+};
+
+// hardcoded for convenience
+pub const STAKE_RENT_EXEMPTION: u64 = 2_282_880;
+
+#[test]
+fn assert_stake_rent_exemption() {
+    assert_eq!(
+        Rent::default().minimum_balance(StakeStateV2::size_of()),
+        STAKE_RENT_EXEMPTION
+    );
+}
+
+/// Resolve all accounts for an instruction, including sysvars and instruction accounts
+///
+/// This function re-serializes the stake history sysvar from mollusk.sysvars.stake_history
+/// every time it's called, ensuring that any updates to the stake history are reflected in the accounts.
+pub fn add_sysvars(
```

Comment:
I'm on the fence with this helper. Is it better to automatically populate needed accounts or have the test writer enumerate them at the site of the instruction builder? Hmm.